README.md

```diff
@@ -1,2 +1,17 @@
 # web
 Web app for checking tmt tests, plans and stories
+# API
+API for checking tmt tests and plans metadata
```

Comment:
Could we add version from the very beginning? It could be optional and default to the latest version available, but let's be prepared for the future.

Reply:
Not sure version of what should we include. Do you mean version of tmt? Could you please elaborate?

Reply:
Version of the API itself.